src/generic/generic-component-language-list/generic-component-language-list.twig

```diff
@@ -39,19 +66,19 @@
             {% endif %}
             <div class="ecl-row">
               <div class="ecl-col-md-6">
-                {% for lang in _languages %}
+                {% for _lang in _languages %}
                   {% if loop.index0 == loop.length // 2 %}
                 </div><div class="ecl-col-md-6">
                   {% endif %}
                   {% include '@ecl/generic-component-button' with {
                     'type': 'link',
-                    'label': lang.label ~ (lang.isActive ? '<span class="ecl-icon ecl-icon--check ecl-u-f-r"></span>'),
-                    'href': lang.href,
+                    'label': _lang.label ~ (_lang.isActive ? '<span class="ecl-icon ecl-icon--check ecl-u-f-r"></span>'),
+                    'href': _lang.href,
                     'is_block': true,
-                    'extra_classes': 'ecl-language-list__button' ~ (lang.isActive ? ' ecl-language-list__button--active'),
+                    'extra_classes': 'ecl-language-list__button' ~ (_lang.isActive ? ' ecl-language-list__button--active'),
```

Comment:
I think `isActive` would be `is_active` to match the rest.

Reply:
you're absolutely right 👍